백엔드 Firebase Function 정의 및 프론트엔드 컴포넌트 수정

hyeok8055 opened the pull request:
- Firebase Functions 구조 및 배포 방법 추가
- FormDialog 컴포넌트에 사업자 등록번호 유효성 검증 및 포맷팅 기능 추가
- LiveAuctionBidding 컴포넌트에서 경매 정보 표시 수정
- ProtectedRoute 컴포넌트에서 인증된 사용자 상태 처리 불필요한 반복 제거
- Home1 페이지에서 경매 데이터 로딩 및 표시 기능 수정
- Post 페이지에서 경매 생성 시 사용자 인증 및 판매자 인증 확인 추가
- 경매 서비스에서 경매 생성 및 조회 로직 수정
- 기타 데이터 속성 정보 통합 진행
- firebase 데이터 연동을 위한 page들 데이터 관련 로직 및 렌더링 조건 등 수정

hyeok8055 committed:
백엔드 Firebase Function 정의 및 프론트엔드 컴포넌트 수정
- Firebase Functions 구조 및 배포 방법 추가
- FormDialog 컴포넌트에 사업자 등록번호 유효성 검증 및 포맷팅 기능 추가
- LiveAuctionBidding 컴포넌트에서 경매 정보 표시 수정
- ProtectedRoute 컴포넌트에서 인증된 사용자 상태 처리 개선
- Home1 페이지에서 경매 데이터 로딩 및 표시 기능 개선
- Post 페이지에서 경매 생성 시 사용자 인증 및 판매자 인증 확인 추가
- 경매 서비스에서 경매 생성 및 조회 로직 개선
- 기타 데이터 속성 정보 통합 진행

ljhyeon commented on frontend/my-app/src/App.jsx:
```diff
@@ -114,7 +114,8 @@ function AppContent() {
     return (
       <Layout description={currentDescription}>
-        <Container disableGutters>
+        // 컨테이너 높이 일괄적으로 100% 적용용
```
주석 기호가 잘못 되어서 화면에 표출됩니다. 삭제하거나 주석 처리해주세요

hyeok8055 replied:
수정 반영 완료

hyeok8055 committed:
feat : 백엔드 경매 상태 동기화 및 프론트엔드 반영
- 경매 상태 변경 시 Firestore와 Realtime Database 동기화 로직 추가
- 경매 자동 활성화 및 종료 기능 수정
- 경매 아이템 컴포넌트에서 상태 처리 수정
- 이미지 업로드 및 검증 로직 구현
- 로그아웃 처리 구현
- 나머지 기타 이것저것 나랑 AI가 건드렸는데 문제 생기면 말해줘유유

hyeok8055 committed:
feat : 백엔드 및 프론트엔드 경매 시스템 수정
- 경매 상태 업데이트 로직 개선: 서울 시간 기준으로 시간 처리 및 로그 추가
- 경매 종료 여부 확인 기능 추가: 프론트엔드에서 경매 종료 시 알림 및 상태 처리 개선
- 경매 참여 버튼 비활성화 및 종료 알림 추가
- 기타 코드 정리 및 최적화

hyeok8055 committed:
feat : 이미지 업로드시 용량 압축기능 추가
